jxls/src/main/java/org/jxls/builder/xls/XlsCommentAreaBuilder.java

```diff
@@ -219,11 +224,104 @@ public static void addCommandMapping(String commandName, Class<? extends Command
         return userAreas;
     }
 
+    private void extractLiterals(CellData cellData, String text, List<String> literalList) {
```

Comment:
How do you think about creating a new class for this parsing?

And writing a testcase for it?

And method return value: List<String>
Is cellData needed?